web/tools.html

```diff
@@ -95,5 +103,44 @@ <h4>Tools</h4>
             })
             .catch(error => console.error('Error loading footer:', error));
         </script>
+
```

Comment:
Should this functionality be moved to one of the other existing js files, or even to its own file entirely?